packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -45,6 +47,8 @@ describe('Deposit ERC20 Token', () => {
     })
 
     it('should bridge ERC-20 successfully', () => {
```

Comment:
maybe we would want to say "to the same address"?

Reply:
Done